<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>839</width>
    <height>525</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MeteoAstrologia v 1.0.115</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="margin">
     <number>3</number>
    </property>
    <item>
     <widget class="QMdiArea" name="mdiArea">
      <property name="viewMode">
       <enum>QMdiArea::TabbedView</enum>
      </property>
      <property name="documentMode">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>839</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArchivo">
    <property name="title">
     <string>Archivo</string>
    </property>
    <addaction name="actionDefinir_Fecha_de_Trabajo"/>
    <addaction name="actionOpciones"/>
    <addaction name="separator"/>
    <addaction name="actionSalir"/>
   </widget>
   <widget class="QMenu" name="menuCarga_de_Datos">
    <property name="title">
     <string>Carga de Datos</string>
    </property>
    <addaction name="actionCarga_de_Dia"/>
    <addaction name="actionCarga_de_Sizigia"/>
    <addaction name="actionModificacion_de_Datos"/>
    <addaction name="actionCargar_datos_RBD_y_SFD"/>
    <addaction name="actionRBD_a_Tiempos_Astrales"/>
    <addaction name="actionTiempos_Astrales_a_NOAA"/>
    <addaction name="actionCarga_en_Batch"/>
    <addaction name="actionCarga_de_Archivos"/>
    <addaction name="actionRecalcular_datos_Mensuales"/>
   </widget>
   <widget class="QMenu" name="menuInformacion_Guardada">
    <property name="title">
     <string>Informacion Guardada</string>
    </property>
    <addaction name="actionVer_Tabla_de_Datos"/>
    <addaction name="actionVer_dias"/>
    <addaction name="actionVer_Datos_Mensuales"/>
   </widget>
   <widget class="QMenu" name="menuListados">
    <property name="title">
     <string>Procesar Datos</string>
    </property>
    <addaction name="actionProceso_Por_Tiempo"/>
    <addaction name="actionProceso_Por_Aspectos"/>
    <addaction name="actionProceso_Por_Ciclos"/>
    <addaction name="actionProceso_por_Posicion_Astral"/>
    <addaction name="actionProcesar_por_Dignidad"/>
    <addaction name="actionProcesar_por_mes_lluvia"/>
   </widget>
   <widget class="QMenu" name="menuExcel">
    <property name="title">
     <string>Excel</string>
    </property>
    <addaction name="actionExportar_por_Sizigias_P_Primordiales"/>
    <addaction name="actionExportar_por_Fechas_Escogidas"/>
    <addaction name="actionCalcular_Pto_Primordiales"/>
   </widget>
   <addaction name="menuArchivo"/>
   <addaction name="menuCarga_de_Datos"/>
   <addaction name="menuInformacion_Guardada"/>
   <addaction name="menuListados"/>
   <addaction name="menuExcel"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionSalir">
   <property name="text">
    <string>Salir</string>
   </property>
  </action>
  <action name="actionCarga_de_Dia">
   <property name="text">
    <string>Carga de Dia</string>
   </property>
  </action>
  <action name="actionCarga_de_Sizigia">
   <property name="text">
    <string>Carga de Sizigia</string>
   </property>
  </action>
  <action name="actionOpciones">
   <property name="text">
    <string>Opciones</string>
   </property>
  </action>
  <action name="actionVer_Tabla_de_Datos">
   <property name="text">
    <string>Ver Tabla de Datos</string>
   </property>
  </action>
  <action name="actionModificacion_de_Datos">
   <property name="text">
    <string>Modificar Datos</string>
   </property>
  </action>
  <action name="actionAspectos">
   <property name="text">
    <string>Aspectos</string>
   </property>
  </action>
  <action name="actionTiempos">
   <property name="text">
    <string>Tiempos</string>
   </property>
  </action>
  <action name="actionProcesar_Datos">
   <property name="text">
    <string>Procesar Datos por Varios</string>
   </property>
  </action>
  <action name="actionBusqueda_Ping_Pong">
   <property name="text">
    <string>Busqueda &quot;Ping-Pong&quot;</string>
   </property>
  </action>
  <action name="actionBusqueda_por_Estado_de_Tiempo">
   <property name="text">
    <string>Busqueda por Estado de Tiempo</string>
   </property>
  </action>
  <action name="actionCarga_en_Batch">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Carga en Batch</string>
   </property>
  </action>
  <action name="actionProceso_Por_Tiempo">
   <property name="text">
    <string>Proceso Por Estado de Tiempo</string>
   </property>
  </action>
  <action name="actionProceso_Por_Aspectos">
   <property name="text">
    <string>Procesar por Aspectos</string>
   </property>
  </action>
  <action name="actionProceso_Por_Ciclos">
   <property name="text">
    <string>Procesar por Ciclos</string>
   </property>
  </action>
  <action name="actionProceso_por_Posicion_Astral">
   <property name="text">
    <string>Procesar por Posicion Astral</string>
   </property>
  </action>
  <action name="actionVer_dias">
   <property name="text">
    <string>Ver dias</string>
   </property>
  </action>
  <action name="actionCarga_de_Archivos">
   <property name="text">
    <string>Carga de Archivos</string>
   </property>
  </action>
  <action name="actionRecalcular_datos_Mensuales">
   <property name="text">
    <string>Recalcular datos Mensuales</string>
   </property>
  </action>
  <action name="actionVer_Datos_Mensuales">
   <property name="text">
    <string>Ver Datos Mensuales</string>
   </property>
  </action>
  <action name="actionProcesar_por_Dignidad">
   <property name="text">
    <string>Procesar por Dignidad</string>
   </property>
  </action>
  <action name="actionProcesar_por_mes_lluvia">
   <property name="text">
    <string>Procesar por mes-lluvia</string>
   </property>
  </action>
  <action name="actionExportar_por_Sizigias_P_Primordiales">
   <property name="text">
    <string>Exportar por Sizigias/P. Primordiales</string>
   </property>
  </action>
  <action name="actionExportar_por_Fechas_Escogidas">
   <property name="text">
    <string>Exportar por Fechas Escogidas</string>
   </property>
  </action>
  <action name="actionCalcular_Pto_Primordiales">
   <property name="text">
    <string>Calcular Pto Primordiales</string>
   </property>
  </action>
  <action name="actionDefinir_Fecha_de_Trabajo">
   <property name="text">
    <string>Definir Fecha de Trabajo</string>
   </property>
  </action>
  <action name="actionCargar_datos_RBD_y_SFD">
   <property name="text">
    <string>Cargar datos RBD y SFD</string>
   </property>
  </action>
  <action name="actionRBD_a_Tiempos_Astrales">
   <property name="text">
    <string>RBD a Tiempos Astrales</string>
   </property>
  </action>
  <action name="actionTiempos_Astrales_a_NOAA">
   <property name="text">
    <string>Tiempos Astrales a NOAA</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
